core/test/dataset_binary_arithmetic_test.cpp

```diff
@@ -527,7 +539,7 @@ TYPED_TEST(DatasetProxyBinaryEqualsOpTest, rhs_Dataset_coord_mismatch) {
 
 TYPED_TEST(DatasetProxyBinaryEqualsOpTest, rhs_Dataset_with_missing_items) {
   auto a = datasetFactory.make();
-  a.setData("extra", makeVariable<double>({}));
+  a.setData("extra", createVariable<double>(Values{double{}}));
```

Comment:
Have no idea. This is done like it is because the compiler picks up `makeVariable<T>(T t)` overload in this case, now create variable does the same. This can be fixed, but as a separate issue.